src/Sentry/Internal/Hub.cs

```diff
@@ -156,6 +163,44 @@ public void BindException(Exception exception, ISpan span)
 
         public SentryTraceHeader? GetTraceHeader() => GetSpan()?.GetTraceHeader();
 
+        public void StartSession()
+        {
+            try
+            {
+                var sessionUpdate = _sessionManager.StartSession();
+                if (sessionUpdate is not null)
+                {
+                    CaptureSession(sessionUpdate);
+                }
```

Comment:
When can it return null here? Should we log Debug?

Reply:
If the `release` is unset for example. It's already logged inside session manager.